qp_shotgun/humann2/humann2.py

```diff
@@ -86,15 +91,32 @@ def generate_humann2_analysis_commands(forward_seqs, reverse_seqs, map_file,
             % ', '.join(sn_by_rp.keys()))
 
     cmds = []
-    params = ['--%s "%s"' % (k, v) for k, v in viewitems(parameters) if v]
+    params = ['--%s "%s"' % (k, v) if v is not True else '--%s' % k
+              for k, v in viewitems(parameters) if v]
     for ffn, fn, s in samples:
+        od = join(out_dir, fn)
+        # just making sure the output directory exists
+        if assure_od and not exists(od):
```

Comment:
Is this a result of humann2 not being executed on travis? otherwise I don't understand why the tests are checking if the directory exists if this function is only creating the command.

i.e. if this is added to cheat on the tests, either there is something wrong on the tests, or there is something wrong on the code.